mapbox/validation.py

```diff
@@ -0,0 +1,31 @@
+
+class MapboxValidationError(ValueError):
+    pass
+
+
+class InvalidPlaceTypeError(MapboxValidationError):
+    pass
+
+
+class InvalidProfileError(MapboxValidationError):
+    pass
+
+
+class InvalidFeatureError(MapboxValidationError):
+    pass
+
+
+class MapboxHTTPError(MapboxValidationError):
+    pass
+
+
+def lat(val):
+    if val is not None and (val < -90 or val > 90):
+        raise MapboxValidationError("Latitude must be between -90 and 90")
+    return val
+
+
+def lon(val):
+    if val is not None and (val < -180 or val > 180):
+        raise MapboxValidationError("Longitude must be between -180 and 180")
+    return val
```

Comment:
> latitudes and longitudes outside these ranges

Yes, in fact even stricter for lat. For the staticmaps API, 

    Error: {"message":"Longitude must be between -180-180."}
    Error: {"message":"Latitude must be between -85.0511-85.0511."}